.circleci/config.yml

```diff
@@ -1,14 +1,14 @@
 jobs:
   build:
-    working_directory: /go/src/github.com/observatorium/deployments
+    working_directory: /go/src/github.com/observatorium/observatorium
     docker:
       - image: quay.io/coreos/jsonnet-ci
     steps:
       - checkout
       - run: make fmt
       - run: make generate-cert
-      - run: make generate tests/manifests
-      - run: git diff --exit-code -- . ':(exclude)tests/manifests/observatorium-xyz-tls-secret.yaml' ':(exclude)tests/manifests/observatorium-xyz-tls-configmap.yaml' ':(exclude)tests/manifests/observatorium-xyz-tls-dex.yaml' ':(exclude)tests/manifests/test-ca-tls.yaml'
+      - run: make generate deployments/tests/manifests
```

Comment:
`configuration` ? (:

Reply:
or will clash with rhobs/configuration repo?

Reply:
I don't think a clash will happen. Also I don't have any opinions about the sub directory name as well. I will change it to `configuration` if that sounds better to you.